.github/workflows/ci.yml

```diff
@@ -5,8 +5,7 @@ on:
     branches:
       - develop
   pull_request:
-    branches:
-      - develop
+    types: [opened, synchronize, reopened]
```

Comment:
AFAICT those types are the default for `pull_request` events. Is there a reason to specify them? Why not simply

```yaml
  pull_request:
```

Reply:
Good to know. Same in all our PRs. I'll remove it everywhere when necessary.